StaticAnalyzer/models.py

```diff
@@ -2,20 +2,20 @@
 # Create your models here.
 
 class RecentScansDB(models.Model):
-    NAME = models.TextField()
+    NAME = models.CharField(max_length=200)
     MD5 = models.CharField(max_length=32)
-    URL = models.TextField()
+    URL = models.URLField()
     TS = models.DateTimeField()
 
 class StaticAnalyzerAndroid(models.Model):
-    TITLE = models.TextField()
-    APP_NAME = models.TextField()
+    TITLE = models.CharField(max_length=50)
+    APP_NAME = models.CharField(max_length=150)

```

Comment:
Again not a good idea to set max limits on things we don't have control on

Reply:
1. Yes, it's in fact not required, It's not used anywhere I guess.
2. They might have, but I can always rename my files APK, IPA or ZIP according to what the OS supports and linux distro supports filename upto 255 chars

Reply:
This name should be the display name of the App, not the file name. Shouldn't it be that long?

Reply:
Display name of the App?  What is that? That field stores the name of the file as it was during the analysis time.

<img width="521" alt="screenshot 2018-10-28 at 12 57 13 am" src="https://user-images.githubusercontent.com/4301109/47608517-8c3d5700-da4c-11e8-9a75-002948c5e150.png">

Reply:
You can keep it to 260